tests/jvm/src/test/resources/metac.expect

```diff
@@ -2377,6 +2458,9 @@ local1 => selfparam self: B
 local2 => selfparam self: B with Int {}
   B => selfs.B#
   Int => scala.Int#
+local3 => selfparam x$1
+local4 => selfparam x$2: B
+  B => selfs.B#
```

Comment:
@olafurpg Can we open a ticket for these guys as well?

Reply:
Done https://github.com/scalameta/scalameta/issues/1662

I didn't observe these were generated from the wildcards, I agree they should not be persisted.